src/c#/main/event/EventProducer.cs

```diff
@@ -63,5 +63,11 @@ public class EventProducer {
             eventRepository.addEvent(pawnDeathEvent);
             Debug.Log("Produced event: " + pawnDeathEvent);
         }
+
+        public void producePawnRelationshipIncreaseEvent(Pawn pawn1, Entity entity2, int increase) {
+            PawnRelationshipIncreaseEvent pawnRelationshipIncreaseEvent = new PawnRelationshipIncreaseEvent(pawn1, entity2, increase);
+            eventRepository.addEvent(pawnRelationshipIncreaseEvent);
+            Debug.Log("Produced event: " + pawnRelationshipIncreaseEvent);
```

Comment:
Curious, I saw you remove a bunch of Debug.LogWarning files in previous files, is this something you want to keep long term? I don't see why not, especially if you compile for production in a way that stops most of this logging. Perhaps add back the logging that was removed earlier?

Reply:
The debug statements I removed were causing spam since the cases were being hit constantly. We may want to consider using a logger with multiple log levels so we can have more control over this. For now I'd like to keep these debug statements removed so we can see when other warning messages come through.

Reply:
A logger with multiple log levels is a good solution, I am happy with you leaving them for now, but heavily recommend creating work that will lead to a Logger being availiable.